src/DirectusImage.tsx

```diff
@@ -3,21 +3,76 @@ import * as React from 'react';
 import { DirectusContext } from './DirectusProvider';
 import { DirectusImageProps } from './types';
 
-export const DirectusImage = ({ apiUrl: propsApiUrl, asset, render, ...params }: DirectusImageProps): JSX.Element => {
+export const DirectusImage = ({
+  apiUrl: propsApiUrl,
+  asset,
+  width,
+  height,
+  quality = 75,
```

Comment:
Why we need to force our default quality here?

Reply:
While it may not be strictly necessary,but it used to be the recommendation by Directus. Reducing data by nearly 80% with minimal image quality loss sounds like a compelling trade-off.

Reply:
I understand but this should be on directus side or, at least, we should put a value that is commonly accepted/used. I see 85 often instead 😅

Reply:
Honestly, my opinion on this is partly based on the example provided on [https://docs.directus.io/reference/files.html#requesting-a-thumbnail](https://docs.directus.io/reference/files.html#requesting-a-thumbnail). I haven't extensively tested the size based on quality, but I believe it's worth considering one of the sizes given in the example. But I'm open for everything in that regard.

Reply:
I'm just saying that we shouldn't force a particular value, if one wants 50 images in a page with the default directus quality (I assume it's 100%)... don't force it to update each component and pass quality="100" just because we decided that 75 is the default or best value 😄 

In addition, Directus may introduce some server-side quality default, in that case it would be impossibile to use `DirectusImage` with the default server-side quality value.

Just my thoughts

Reply:
You are right, I will remove it.
Maybe we could add DirectusImage prop defaults to the Provider in the future.

Reply:
Yes, giving the ability to provide some defaults in the Provider is a nice idea 👍 (of course, only those settings that make sense)

Reply:
"Yes, I was thinking about something like:
- `presetKey` type: This would provide type safety when adding preset keys.
- `presetKey` only mode: Enabling this mode would disable dynamic options, which could potentially be a security risk (refer to [File Storage](https://docs.directus.io/user-guide/cloud/project-settings.html#files-storage)).
- Defaults for all dynamic props: Adding defaults would enhance the user experience when using dynamic props.